website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -670,6 +672,20 @@ A `windows_profile` block supports the following:
 
 ---
 
+A `http_proxy_config` block supports the following:
+
+* `http_proxy` - (Optional) The proxy address to be used when communicating over HTTP.
+
+* `https_proxy` - (Optional) The proxy address to be used when communicating over HTTPS.
+
+* `no_proxy` - (Optional) The list of domains that will not use the proxy for communication.
+
+-> **Note:** If you specify the `vnet_subnet_id`, be sure to include the CIDR in the no_proxy list.
+
+* `trusted_ca` - (Optional) The base64 encoded alternative CA certificate conten in PEM format.
```

Comment:
Minor typo fix. Additionally, if you wanted to validate this, you could un-base64 it and validate that it does contain a TLS public certificate with the CA flag set.

```suggestion
* `trusted_ca` - (Optional) The base64 encoded alternative CA certificate content in PEM format.
```

Reply:
Typo should be fixed now, thanks!